draft-ftbs-rats-msg-wrap.md

```diff
@@ -282,6 +282,44 @@ This document does not make any requests to IANA.
 {::include cddl/rfc9193.cddl}
 ~~~
 
+# Registering and Using CMWs
+
+{{fig-howto-cmw}} describes the registration preconditions for using
+CMWs in either array or CBOR tag forms.
+
+~~~ aasvg
+        .-------------.    .---------.
+       | Reuse EAT     |  | Register  |
```

Comment:
note that CoRIM uses the same scheme (i.e., base + profile) as EAT

Reply:
done in [1326297](https://github.com/thomas-fossati/draft-ftbs-rats-msg-wrap/pull/21/commits/1326297a681d2012587d9ce821b349d2007cc50f)